1.0/docs/devguide/styling.md

```diff
@@ -8,11 +8,27 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-
 Polymer uses [Shadow DOM styling
 rules](http://www.html5rocks.com/en/tutorials/webcomponents/shadowdom-201/) for
-providing scoped styling of the element's local DOM.  Scoped styles should be
-provided via `<style>` tags placed inside the element's local DOM `<template>`.
+providing scoped styling of the element's local DOM. This allows elements to
+create style rules that will only apply to the element's light DOM or local
```

Comment:
2 points here:

-   "light DOM" is not correct. This should probably be "the element's local DOM and distributed children." 

    Explanation: there are only two, specific places where scoped rules can touch stuff outside the element's local DOM: you can style the host element itself, with `:host`, and you can style distributed children, with `::content`. The distinction between light DOM children and distributed children is subtle, but important. If you don't have a `<content>` tag, you can't style your light DOM children, for instance. If you use `<content select=`, you can't style anything that doesn't match one of your selectors, because it doesn't get distributed.

-   The CSS custom properties shouldn't be mentioned in the same sentence. They're not related to the shadow DOM spec, they're a completely different specification that happens to be useful for styling shadow DOM. (We also talk about "the shady DOM shim" and "the custom properties shim" as separate entities.)